templates/pcab/image.tpl.html

```diff
@@ -0,0 +1,4 @@
+<div class="images"><img src='<%= file.url %>' alt='<%= description %>'>
+    <span class="image-caption"><%= caption %>
+    </span>
```

Comment:
Either put `</span>` at the end of the previous row, or break the entire thing down to three rows.